### PR TITLE
update to support agent 1.14.2

### PR DESCRIPTION
**Pull request by adnxn**
* Cache Agent version 1.14.2

### CODE REVIEW

**Comment by nmeyerhans** [comment]
@aaithal Nope, that file no longer needs to change. Values are injected at runtime as of PR #91

**Comment by aaithal** [comment]
@nmeyerhans oh cool, nice.

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 1]
```diff
@@ -149,6 +149,9 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Fri May 26 2017 Adnan Khan <adnkha@amazon.com> - 1.14.2-1
+- Cache Agent version 1.14.2
+- Add functionality for region based agent updates
```
Comment (nmeyerhans) [inline packaging/amazon-linux-ami/ecs-init.spec]: There should be an entry here about running agent with userns=host when Docker is running with userns-remap enabled.

Comment (adnxn) [inline packaging/amazon-linux-ami/ecs-init.spec]: gotcha. also i just realised it shouldn't have that region based agent updates entry for this release.

**Comment by nmeyerhans** [comment]
:ship: LGTM

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 2]
```diff
@@ -149,6 +149,9 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Thu Jun 1 2017 Adnan Khan <adnkha@amazon.com> - 1.14.2-1
+- Cache Agent version 1.14.2
+- Add functionality for running agent with userns=host when Docker has userns-remap enabled
```
Comment (aaithal) [inline packaging/amazon-linux-ami/ecs-init.spec]: I think we're missing an entry here and in the spec file for the docker `17.03.x-ce` upgrade.

Comment (samuelkarp) [inline packaging/amazon-linux-ami/ecs-init.spec]: @aaithal is correct, see line 30.

Comment (adnxn) [inline packaging/amazon-linux-ami/ecs-init.spec]: fixed both things.

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 3]
```diff
@@ -149,6 +149,10 @@ if [ -e %{running_semaphore} ]; then
...
+- Add support for Docker 17.03.1ce
```
Comment (aaithal) [inline packaging/amazon-linux-ami/ecs-init.spec]: Should we say `ce` here if that's not specified in the spec file?

Comment (adnxn) [inline packaging/amazon-linux-ami/ecs-init.spec]: i'm not sure tbh. i figured if the user checks ```docker version``` they would get the "-ce" bit as well.

Comment (aaithal) [inline packaging/amazon-linux-ami/ecs-init.spec]: let's be consistent. If we're not calling it 'ce' in the spec file, we should leave that bit here too i think.

Comment (adnxn) [inline packaging/amazon-linux-ami/ecs-init.spec]: turns out we *have* to call it "ce" in the spec file, otherwise it makes rpm versioning sad.

**Pull request merged**